parity/dir.rs

```diff
@@ -73,6 +73,16 @@ impl Directories {
 		dir.push("ipc");
 		dir
 	}
+
+	/// Get user defaults path
+	pub fn user_defaults_path(&self, fork_name: &Option<String>) -> PathBuf {
```

Comment:
But it's also using `genesis_hash` - User defaults is not property of the `fork` it's a property of specific database.
Maybe it should be placed inside `<genesis_hash>` directory like `snapshots`:
https://github.com/ethcore/parity/blob/1c19a807d9996e30e9e2f772c5a7f73e7e80ff07/ethcore/src/snapshot/service.rs#L221

It would then make it clear that it's user defaults for specific `genesis+fork`